cli/azd/pkg/azdo/build_policy.go

```diff
@@ -97,3 +143,68 @@ func CreateBuildPolicy(
 
 	return nil
 }
+
+func newLocalClient(ctx context.Context, connection *azuredevops.Connection) (*clientImpl, error) {
+	client, err := connection.GetClientByResourceAreaId(ctx, uuid.MustParse("fb13a388-40dd-4a04-b530-013a739c72ef"))
+	if err != nil {
+		return nil, err
+	}
+	return &clientImpl{
+		Client: *client,
+	}, nil
+}
+
+type clientImpl struct {
+	Client azuredevops.Client
+}
+
+// local implementation for GetPolicyConfigurations
+// The implementation from the policy client is broken because it does not support taking a continuation token
+// see: https://github.com/microsoft/azure-devops-go-api/issues/156
+func (client *clientImpl) getPolicyConfigurations(
+	ctx context.Context, args getPolicyConfigurationsArgs) (*getPolicyConfigurationsResponseValue, error) {
+	routeValues := make(map[string]string)
+	if args.Project == nil || *args.Project == "" {
+		return nil, &azuredevops.ArgumentNilOrEmptyError{ArgumentName: "args.Project"}
+	}
+	routeValues["project"] = *args.Project
+
+	queryParams := url.Values{}
+	if args.Scope != nil {
+		queryParams.Add("scope", *args.Scope)
+	}
+	if args.PolicyType != nil {
+		queryParams.Add("policyType", (*args.PolicyType).String())
+	}
+	if args.ContinuationToken != "" {
+		queryParams.Add("continuationToken", args.ContinuationToken)
+	}
+	locationId, _ := uuid.Parse("dad91cbe-d183-45f8-9c6e-9c1164472121")
```

Comment:
Why are we hard coding coding this location id ?

Reply:
that's code from the SDK for this API.

The only difference in this function to the SDK function is:

```go
if args.ContinuationToken != "" {
		queryParams.Add("continuationToken", args.ContinuationToken)
	}
```